pkg/service/redisstore.go

```diff
@@ -242,21 +242,29 @@ func (s *RedisStore) LockRoom(_ context.Context, roomName livekit.RoomName, dura
 	return "", ErrRoomLockFailed
 }
 
-func (s *RedisStore) UnlockRoom(_ context.Context, roomName livekit.RoomName, uid string) error {
+func (s *RedisStore) UnlockRoom(ctx context.Context, roomName livekit.RoomName, uid string) error {
 	key := RoomLockPrefix + string(roomName)
+	unlockScript := `if redis.call("get", KEYS[1]) == ARGV[1] then
+						return redis.call("del", KEYS[1])
+					 else return 0 
+					 end`
+
+	res, err := redis.NewScript(unlockScript).Run(ctx, s.rc, []string{key}, uid).Result()
```

Comment:
can we store `redis.NewScript(unlockScript)` as a struct member? instead of having to reencode the script each time.

Reply:
done